src/converter.ts

```diff
@@ -291,6 +291,11 @@ export class Converter {
     })
     pptx.layout = layoutName
 
+    // TODO: Revert the workaround for regression about custom layout in PptxGenJS v3.3.0
+    // https://github.com/gitbrent/PptxGenJS/issues/826
+    pptx.presLayout['width'] = pptx.presLayout['_sizeW']
+    pptx.presLayout['height'] = pptx.presLayout['_sizeH']
```

Comment:
...but still requires another hack to deal with a regression about the custom slide layout in v3.3.0.
https://github.com/gitbrent/PptxGenJS/issues/826